docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
```

Comment:
may be say that it's "Node Settings"? May be some people will think that you can set that when you define the pipeline?